app/dashboard/models.py

```diff
@@ -4791,7 +4794,8 @@ def psave_hackathonevent(sender, instance, **kwargs):
                 "visible_to":None,
                 'img_url': instance.logo.url if instance.logo else None,
             }
-            )
```

Comment:
please dont make arbitrary style changes like this as it makes your PRs much harder to review

Reply:
I am sure when the change was introduced. I shall revert it.

Reply:
@danlipert I am sure when the change was introduced. I shall revert it.